website/src/app/team/page.tsx

```diff
@@ -0,0 +1,279 @@
+              </span>
+              <p className="mt-3 mb-4 max-w-sm font-light text-gray-500 dark:text-gray-400">
+                {/* TODO: bio */}
+              </p>
+              <ul className="flex justify-center space-x-4 sm:justify-start">
+                <li>
+                  <TwitterIcon url="https://twitter.com/franlovebloom" />
+                </li>
+                <li>
+                  <GitHubIcon url="https://github.com/francesca64" />
+                </li>
+                <li>
+                  <LinkedInIcon url="https://www.linkedin.com/in/francesca-lovebloom/" />
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div className="flex flex-col items-center py-8 sm:flex-row">
+            <Image
+              width={144}
+              height={144}
+              className="mx-auto mb-4 w-36 h-36 rounded-full sm:ml-0 sm:mr-6"
+              src="/images/avatars/brian.png"
+              alt="Brian Avatar"
+            />
+            <div className="text-center sm:text-left">
+              <h3 className="justify-center sm:justify-start text-xl font-bold tracking-tight text-gray-900 dark:text-white">
+                Brian Manifold
+              </h3>
+              <span className="text-gray-500 dark:text-gray-400">
+                Senior Fullstack Engineer
+              </span>
+              <p className="mt-3 mb-4 max-w-sm font-light text-gray-500 dark:text-gray-400">
+                {/* TODO: bio */}
+              </p>
+              <ul className="flex justify-center space-x-4 sm:justify-start">
+                <li>
+                  <GitHubIcon url="https://github.com/bmanifold" />
+                </li>
+                <li>
+                  <LinkedInIcon url="https://www.linkedin.com/in/brian-manifold-536a0a3a/" />
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div className="text-gray-500 sm:text-lg dark:text-gray-400">
+            <h3 className="justify-center py-8 text-xl tracking-tight font-bold text-gray-900 dark:text-white">
+              ADVISORS & CONSULTANTS
+            </h3>
+          </div>
+          <div className="flex flex-col items-center py-8 sm:flex-row">
+            <Image
+              width={144}
+              height={144}
+              className="mx-auto mb-4 w-36 h-36 rounded-full sm:ml-0 sm:mr-6"
+              src="/images/avatars/blake.jpeg"
+              alt="Blake Avatar"
+            />
+            <div className="text-center sm:text-left">
+              <h3 className="justify-center sm:justify-start text-xl font-bold tracking-tight text-gray-900 dark:text-white">
+                Blake Hitchcock
+              </h3>
+              <span className="text-gray-500 dark:text-gray-400">
+                Technical Advisor
+              </span>
+              <p className="mt-3 mb-4 max-w-sm font-light text-gray-500 dark:text-gray-400">
+                {/* TODO: bio */}
```

Comment:
@rbhitchcock Do you have a short bio you'd like to include here on our new website?